WarriorDeliveryWebsite/src/Theme.css

```diff
@@ -99,22 +99,23 @@ nav {
 	padding: 0px;
 	display: block;
 	clear: both;
+	margin: 1vw 0vw 1vw 0vw;
 }
 
 nav a:link{
 	color: #000000;
 	background-color: #B80C09;
-	font-size: 1.5em;
+	font-size: 1.75em;
 	border: none;
 	border-radius: 0px;
-	padding: 10px;
+	padding: 1vw;
```

Comment:
leaving comments on what these do exactly will help. The values are not obvious at first sight.